File: seunggon/week2/easy.py
def solution(babbling):
    answer = 0
    pron = ["aya", "ye", "woo", "ma"]

    for word in babbling:
        valid = True
       
        for i in pron:
            if i * 2 in word:  # "ayaaya" 확인
                valid = False
                break
        
        if not valid:
            continue
            
        for j in pron:
            word = word.replace(p, " ") # 발음 가능한 부분 -> 공백 
        
        if word.strip() == "":
            answer += 1
            
    return answer

File: juha/week2/easy.js
function solution(babbling) {
  const wordList = ["aya", "ye", "woo", "ma"];
  let count = 0;

  for (let babbel of babbling) {
    let valid = true;

    for (let s of wordList) {
      if (babbel.includes(s + s)) {
        valid = false;
        break;
      }
    }

    if (!valid) continue;

    let currentWord = babbel;
    for (let s of wordList) {
      currentWord = currentWord.split(s).join(" ");
    }

    if (currentWord.trim().length === 0) {
      count++;
    }
  }

  return count;
}
